src/site/stages/build/drupal/graphql/nodeSupportResourcesDetailPage.graphql.js

```diff
@@ -0,0 +1,52 @@
+const entityElementsFromPages = require('./entityElementsForPages.graphql');
+
+const fragment = `
+fragment nodeSupportResourcesDetailPage on NodeSupportResourcesDetailPage {
```

Comment:
We kinda started a pattern to preface node queries with "node" but I think we only did that in the beginning so the "node QA" was distinguished from the "QA paragraph." It's up to you to continue that pattern of using "node" in the beginning. It's kinda nice I think

Reply:
I like it, too! 🙂 It's the exact name of the entity, which I prefer over a short name